### PR TITLE
lnd: show debug level at startup

### PR DESCRIPTION
**Pull request by MrManPew**
This will help to verify the debuglevel that is used

### CODE REVIEW

File: lnd.go [hunk 1]
```diff
@@ -200,9 +200,9 @@ func Main(cfg *Config, lisCfg ListenerCfg, shutdownChan <-chan struct{}) error {
 	}()
 
 	// Show version at startup.
-	ltndLog.Infof("Version: %s commit=%s, build=%s, logging=%s",
+	ltndLog.Infof("Version: %s commit=%s, build=%s, logging=%s debuglevel=%s",
```
Comment (bhandras) [inline lnd.go:203]: Please make sure to follow the contribution guidelines for commit messages.
Also the short commit summary should be something like this: 
`lnd: show debug level at startup`
 (We usually add the module name to the commit).

**Review by bhandras — CHANGES_REQUESTED** [review]

File: lnd.go (hunk 1 above)
Comment (guggero) [reply lnd.go:203]: Also we should probably add a comma before the `debuglevel` to follow the existing pattern.

Comment (MrManPew) [reply lnd.go:203]: Are you sure we need a comma, there is not comma between the Version  and commit ? I assumed that logging and debuglevel are similar.

Comment (MrManPew) [reply lnd.go]: Added comma, changed the commit message (took this as an example:
https://github.com/lightningnetwork/lnd/pull/4649)

**Review by bhandras — APPROVED** [review]

**Review by cfromknecht — APPROVED** [review]
LGTM 🥂

**Pull request merged**